interfaces/repositories.go

```diff
@@ -73,6 +77,38 @@ func (repo *DbUserRepo) FindById(id int) (usecases.User, error) {
 	return user, nil
 }
 
+func (repo *DbUserRepo) Count() int {
+	row, _ := repo.dbHandler.Query("SELECT user_name FROM users")
+	count := 0
+	for row.Next() {
+		count++
+	}
+	return count
+}
+
+func (repo *DbUserRepo) NameExisted(userName string) bool {
+	row, _ := repo.dbHandler.Query(`SELECT user_name FROM users
+		WHERE user_name=$1 LIMIT 1`, userName)
+	return row.Next()
+}
+
+func (repo *DbUserRepo) StoreInfo(user usecases.User, info string) error {
+	_, err := repo.dbHandler.Execute(`UPDATE users SET personal_info=$1
+		WHERE id=$2`, info, user.Id)
+	return err
+}
+
+func (repo *DbUserRepo) LoadInfo(user usecases.User) (string, error) {
+	row, err := repo.dbHandler.Query(`SELECT personal_info FROM users WHERE id=$1`, user.Id)
+	if err != nil {
+		return "", nil
```

Comment:
+func (repo *DbUserRepo) NameExisted(userName string) bool {
+	row, _ := repo.dbHandler.Query(`SELECT user_name FROM users
+		WHERE user_name=$1 LIMIT 1`, userName)
+	return row.Next()
+}

Are you referring to this part?